iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisClient.java

```diff
@@ -212,46 +218,39 @@ public Action handleAttemptFailure(Event event) {
   }
 
   // This policy is used to raft configuration change
+  //
   private static class RatisEndlessRetryPolicy implements RetryPolicy {
 
     private static final Logger logger = LoggerFactory.getLogger(RatisEndlessRetryPolicy.class);
-    private static final RetryPolicy defaultPolicy;
-
-    static {
-      String str = "";
-      // 50, 500ms, 40, 1000ms, 30, 1500ms, 20, 2000ms, 10, 2500ms
-      int basicRetry = 50;
-      int basicSleep = 500;
-      for (int i = 0; i < 5; i++) {
-        str += basicRetry + "," + basicSleep + ",";
-        basicRetry -= 10;
-        basicSleep += 500;
-      }
+    private final RetryPolicy defaultPolicy;
 
+    RatisEndlessRetryPolicy() {
+      // about 1 hour wait Time.
       defaultPolicy =
-          MultipleLinearRandomRetry.parseCommaSeparated(str.substring(0, str.length() - 1));
+          RetryPolicies.retryForeverWithSleep(TimeDuration.valueOf(2, TimeUnit.SECONDS));
     }
```

Comment:
1. Isn't retry every 2 seconds too frequently ?
2. Why still // about 1 hour wait Time.

Reply:
1. if reply return false, then resend request after 2s.  Instead of sending it every 2 seconds

sorry, I forget delete comment